### PR TITLE
Dont keep the fake runner around as it fails validation when preparing release archives

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: sky/build/sdk_xcode_harness/FlutterApplication.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -236,7 +236,7 @@
 				9E07CF971BE8280A00BCD8DE /* Frameworks */,
 				9E07CF981BE8280A00BCD8DE /* Resources */,
 				9E07CFB91BE82D2600BCD8DE /* Embed Frameworks */,
-				9E40465C1C1B6DEC00A4B87C /* Copy Flutter Runner Executable And Resources */,
+				9E40465C1C1B6DEC00A4B87C /* Replace Fake Runner with Acutal Flutter Runner */,
```
Comment (Hixie) [inline sky/build/sdk_xcode_harness/FlutterApplication.xcodeproj/project.pbxproj]: actual typo

Comment (Hixie) [inline sky/build/sdk_xcode_harness/FlutterApplication.xcodeproj/project.pbxproj]: x3

**Comment by Hixie** [comment]
rslgtm

**Comment by chinmaygarde** [comment]
Derp. Rebased.

**Pull request merged**